src/index.js

```diff
@@ -3,7 +3,7 @@ import fs from 'fs';
 import { resolve, relative, dirname, basename, extname } from 'path';
 import chalk from 'chalk';
 import { map, series } from 'asyncro';
-import glob from 'glob';
+import glob from 'tiny-glob/sync';
```

Comment:
any reason why did u chose to use sync version?

Reply:
because it was always sync